src/search_request.c

```diff
@@ -48,6 +49,34 @@ RSSearchRequest *ParseRequest(RedisSearchCtx *ctx, RedisModuleString **argv, int
     req->slop = __INT_MAX__;
   }
 
+  int sumIdx;
```

Comment:
I've redone all stateful parsing in another PR, but that's for after this is merged.